spec/models/allocation_list_spec.rb

```diff
@@ -5,7 +5,7 @@
     AllocationService.create_allocation(
       nomis_staff_id: 485_595,
       nomis_offender_id: 'G2911GD',
-      created_by: 'Test User',
+      created_by: 'PK000223',
```

Comment:
No, the created_by_name is set when the allocation is created, otherwise we'll end up doing lots of work every time we want to create an allocation.  The created_by is required though, and should be a user name.